story_services_test.py

```diff
@@ -0,0 +1,2322 @@
+
```

Comment:
Please add your tests to the existing [`core/domain/story_services_test.py`](https://github.com/oppia/oppia/blob/develop/core/domain/story_services_test.py) file instead